shippable.resources.yml

```diff
@@ -2,7 +2,7 @@ resources:
 # resources for the provision-gke-kubernetes-cluster app
   - name: dmsk_img
     type: image
-    integration: drship_gcr
+    integration: drship_gcr    #replace with your Google Cloud integration name
```

Comment:
we need to change the integration name here to the new google cloud integration. Not sure what the name should be so havent' changed till yet